packages/rich-text/src/component/index.js

```diff
@@ -1024,6 +1041,8 @@ class RichText extends Component {
 				onKeyUp={ this.onSelectionChange }
 				onMouseUp={ this.onSelectionChange }
 				onTouchEnd={ this.onSelectionChange }
+				contentEditable
```

Comment:
Technically we previously applied props such that someone could `contentEditable`. I don't think that's a use case we want to support though 😄

Reply:
Right. I don't think we should support that for now. We might in the future, maybe through a `disabled` or `readonly` prop. But probably not by directly setting `contentEditable.